### ISSUE TITLE
watermark does not support text

### ISSUE THREAD

**Issue by zhanghao1949** [cshum/imagor#431]
First of all, thank you for maintaining Imagor.

I am currently using the watermark feature of imagor and I have found that it does not support text watermarks. Do you have any plans to implement this feature?

**Comment by cshum** [comment]
Do you mean the "label" filter?

https://github.com/cshum/imagor/blob/master/README.md#filters

>  `label(text, x, y, size, color[, alpha[, font]])` adds a text label to the image. It can be positioned inside the image with the alignment specified, color and transparency support

**Comment by zhanghao1949** [comment]
@cshum Thank you for your reply. I have tried "label(test,center,center,100,red,0)", but it can only draw text once in a fixed position. The effect I want to achieve is to repeatedly fill the entire image with text. What parameters should I use to achieve this effect?

**Comment by zhanghao1949** [comment]
@cshum Thank you for your reply. I have tried "label(test,center,center,100,red,0)", but it can only draw text once in a fixed position. The effect I want to achieve is to repeatedly fill the entire image with text. 
<img width="667" alt="test" src="https://github.com/cshum/imagor/assets/49609241/43b6628b-044a-4146-b1ad-642f77c29bc6">
What parameters should I use to achieve this effect?

**Comment by duffn** [comment]
I don't think this is possible with `label`, but you could make your text as an image and use `watermark` to achieve your desired results. 

http://localhost:8000/unsafe/filters:watermark(https://blog.fmctraining.com/hubfs/putting%20an%20image%20in%20text%20photoshop.png,repeat,repeat,40,40,0)/https://raw.githubusercontent.com/cshum/imagor/master/testdata/gopher.png

![gopher](https://github.com/cshum/imagor/assets/3457341/d2d00c2a-b27f-4007-9c96-e4db08159de9)
